amethyst_renderer/src/mouse.rs

```diff
@@ -1,34 +1,38 @@
 //! Util functions that change how the mouse moves and looks.
 
 use super::WindowMessages;
-use winit::{CursorState, MouseCursor};
+use winit::MouseCursor;
 
 /// Hide the cursor, so it's invisible while playing. Can't be used at the same time as grab_cursor.
 pub fn hide_cursor(msg: &mut WindowMessages) {
-    change_cursor_state(msg, CursorState::Hide);
+    msg.send_command(|win| {
+        win.hide_cursor(true);
+    });
```

Comment:
You say you're not sure this code is correct. Have you tested it? (with the other instances of `send_command`)

Reply:
I have not tested it because the examples don't seem to run on Linux (they can't find the config RON files). I can test them tonight on some examples. The code should work though.

Reply:
Rebase on develop, it was a bug that is now solved.

Reply:
Oh wait it's actually not solved yet, the merge failed. Let me fix that.

Reply:
Okay it's fixed now, you can try again after rebasing.

Reply:
Nope
```
cargo run --example fly_camera
...
←[37m[DEBUG][amethyst_assets::loader] "PREFAB": Loading asset "prefab/fly_camera.ron" with format "Ron" from source "[default source]" (handle id: Handle { id: 0, marker: PhantomData })←[0m
thread 'main' panicked at 'Tried to fetch a resource, but the resource does not exist.
Try adding the resource by inserting it manually or using the `setup` method.
You can get the type name of the missing resource by enabling `shred`'s `nightly` feature', C:\Users\Victor\.cargo\registry\src\github.com-1ecc6299db9ec823\shred-0.7.0\src\res\mod.rs:215:48
note: Run with `RUST_BACKTRACE=1` for a backtrace.
error: process didn't exit successfully: `target\debug\examples\fly_camera.exe` (exit code: 101)
```

Reply:
The error might not be related to this pr. We got a report of this example not working because of Named